### ISSUE TITLE
Class "IntlDateFormatter" not found -php 8.2

### ISSUE THREAD

**Issue by gtbu** [gtbu/Typesetter-5.3-p8#24]
Under php 8.2 ,  a click at the **revision-history-button** gives 

Class "IntlDateFormatter" not found  - in ....\thirdparty\time\strftime.php     on line: 61 (54)  - 

* You can for this use the strftime.php-version from https://gist.github.com/bohwaz/42fc223031e2b2dd2585aab159a20f30

and add behind the first line

namespace PHP81_BC;

  use DateTime;
  use DateTimeZone;
  use DateTimeInterface;
  use Exception;
  use IntlDateFormatter;
  use IntlGregorianCalendar;
  use InvalidArgumentException;

**Comment by TheWitness** [comment]
You should ensure that php-intl is installed.  Run `php -m` and look for `Intl`

**Comment by gtbu** [comment]
Yes - but intl is nowadays a standard - except under Xampp, where You sholud activate intl and zip in the php.ini

**Comment by TheWitness** [comment]
The INTL library can not handle common sprintf like:

```php
strftime("The rain in Spain falls on the %M'th day of %Y\n");
```

Why I say they messed up.

**Comment by gtbu** [comment]
Some common mistakes when using sprintf include forgetting to allocate enough space for the output, passing invalid pointers to the function, or providing incorrect format specifiers. These errors can lead to unexpected behavior, such as truncated strings or memory corruption. 
To avoid these issues one sholud consider using safer alternatives to sprintf, such as the newer snprintf function, which allows  to specify the maximum number of characters to write to the output buffer. This helps prevent potential buffer overflow vulnerabilities and ensures that your code behaves predictably.

**Comment by gtbu** [comment]
P.S. In Typesetter 5.3 is now a new more comfortable intltime. 
But there is also https://github.com/moment/luxon which you can use instead in themes
